operator/v1/types_network.go

```diff
@@ -295,6 +295,13 @@ type KuryrConfig struct {
 	// +optional
 	EnablePortPoolsPrepopulation bool `json:"enablePortPoolsPrepopulation,omitempty"`
 
+	// mtu is the MTU to use for the Pods Network.
+	// This must be smaller than the Nodes Network MTU.
+	// If unset the Nodes Network MTU is used.
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	MTU *uint32 `json:"mtu,omitempty"`
```

Comment:
Does `nil` value make sense from our perspective? We could treat `0` as the default value, right? I see it's trying to be consistent with other *SDNConfig structs, so I guess I'm okay with this.

Reply:
yup, I believe it makes sense to be an optional setting, even when Provider Network is used. If issues are hit the user will have the ability to configure it with this setting.